### PR TITLE
Update to 8.6.6

### PR DESCRIPTION
**Pull request by jjhelmus**
Remove python dependency on windows platform.

### CODE REVIEW

**Comment by jjhelmus** [comment]
Thanks to @msarahan for this inspiration for this in PR #1

**Comment by jakirkham** [comment]
This looks very interesting. So I don't know if this would work here, but maybe it is worth a try. `conda-smithy` has support for matrices. Thus one could do something like this. Though it would also affect the other CIs. Not sure if there is a way to change that last point ATM.

```
matrix:
  - [[VC_VERSION, 9]]
  - [[VC_VERSION, 10]]
  - [[VC_VERSION, 14]]
```

cc @pelson

**Comment by msarahan** [comment]
That's a good idea @jakirkham.  You still need to add that variable as a pass-through variable in meta.yaml - or in conda-build if it proves to be a good standard, but this may be a good bridge to the build configuration stuff.

**Comment by jjhelmus** [comment]
Once this is merged and Windows 8.6`tk` packages are available that do not depend on python it looks like it should possible to build Python on Windows with external libraries as suggested by @msarahan in conda-forge/python-feedstock#19.  

See [this build]( https://ci.appveyor.com/project/JonathanHelmus/python-feedstock/build/1.0.111) for an example, just don't look into the commit log of that branch... it is messy.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,21 +12,17 @@ build:
...
-    - vc10    # [win and py34]
-    - vc14    # [win and py35]
+    - vc{{ VC_VERSION }}   # [win]
 
 requirements:
   build:
-    - python         # [win]
     - curl           # [win]
     - 7za            # [win]
+    - vc             # [win]
```
Comment (msarahan) [inline recipe/meta.yaml]: I recommend including the {{ VC_VERSION }} as a version pin in this line and below.  It seems to be working fine without it, but I just don't trust these things...

Happy to merge if you fix this, or if you tell me that you don't want to.

**Comment by jjhelmus** [comment]
@msarahan Agreed, I would rather be explicit with the vc version.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,9 +18,9 @@ requirements:
   build:
     - curl           # [win]
     - 7za            # [win]
-    - vc             # [win]
+    - vc{{ VC_VERSION }}   # [win]
```
Comment (msarahan) [inline recipe/meta.yaml]: Need a space here - it's a version spec, not part of the package name.

Comment (msarahan) [inline recipe/meta.yaml]: Somewhat confusingly, the feature has no space - it is different from the package (only ```vc``` - but with associated version)

**Comment by jjhelmus** [comment]
Corrected the `vc` package to include the version specification.  The `vc` package is a bit odd to work with but VERY helpful.

**Comment by msarahan** [comment]
Thanks @jjhelmus.  Merging.

**Pull request merged**